### PR TITLE
[TACHYON-1559] Add unit test for tachyon.client.ClientUtils.getRandomNonNegativeLong()

### PR DESCRIPTION
**Pull request by sisirkoppaka**
https://tachyon.atlassian.net/projects/TACHYON/issues/TACHYON-1559

### CODE REVIEW

File: clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.Test;
+import org.junit.Assert;
+
+/**
+ * Tests {@link ClientUtils}.
+ */
+public final class ClientUtilsTest {
+
+  /**
+   * Tests if output of getRandomNonNegativeLong from {@link ClientUtils} is non-negative.
```
Comment (aaudiber) [inline clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java]: could instead do `{@link ClientUtils#getRandomNonNegativeLong()}` to defend against IDE refactors

File: clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+   */
+  @Test
+  public void getRandomNonNegativeLongTest() throws Exception {
+    Assert.assertTrue(ClientUtils.getRandomNonNegativeLong() > 0);
```
Comment (aaudiber) [inline clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java]: maybe also check that two random values are different?

File: clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
```
Comment (aaudiber) [inline clients/unshaded/src/test/java/tachyon/client/ClientUtilsTest.java]: NIT: we usually don't put a blank line at the end of the class

**Comment by aaudiber** [comment]
@sisirkoppaka Thanks for adding this unit test! I've left a couple comments

**Comment by sisirkoppaka** [comment]
@aaudiber Thanks for the comments! Pushed changes addressing the issues above.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**